torch/csrc/jit/script/concrete_module_type.cpp

```diff
@@ -72,7 +72,7 @@ ConcreteModuleType::ConcreteModuleType(ConcreteModuleTypeBuilder data)
 bool operator==(
     const ConcreteModuleTypeBuilder::ModuleInfo& lhs,
     const ConcreteModuleTypeBuilder::ModuleInfo& rhs) {
-  return lhs.meta_->equals(*rhs.meta_);
+  return lhs.name_ == rhs.name_ && lhs.meta_->equals(*rhs.meta_);
```

Comment:
Is this qualified name or just name ? If it's just `Foo`, then this won't work for modules in different namespaces.

Reply:
This is name of the attribute, like `foo` in `self.foo`.